app/views/research_sessions/preview.html.erb

```diff
@@ -136,6 +136,20 @@
     </dl>
   </section>
 
+  <section>
+    <h3 class="subtitle-small" id="concerns">Decide if you want <%= 'the child in your care ' if @research_session.unable_to_consent?  %>to take part</h3>
+
+    <p>If you are happy <%= 'for the child in your care ' if @research_session.unable_to_consent? %>to take part, we will ask you to sign a consent form. This will ask you to confirm:</p>
```

Comment:
Line's a little long here, Mr. @joelcarr. Astonishingly it is also L142, which is the same line @kittysquee had to split earlier. I think we might have to mandate that this file only goes up to 141 lines since L142 is clearly problematic.